docs/compute/decentralized_live_migration.md

```diff
@@ -0,0 +1,248 @@
+spec:
+  runStrategy: Always
+  template:
+    metadata:
+  	  creationTimestamp: null
+    spec:
+  	  architecture: amd64
+  	  domain:
+    	    devices:
+    	    autoattachGraphicsDevice: false
+    	  disks:
+    	  - disk:
+             bus: virtio
+      	    name: disk0
+    	  interfaces:
+    	  - masquerade: {}
+      	    name: default
+    	  machine:
+      	    type: q35
+    	  resources:
+      	    requests:
+             memory: 128Mi
+    networks:
+    - name: default
+  	  pod: {}
+    volumes:
+  	  - dataVolume:
+      	    name: example-decentralized-lm-vm-dv
+    	  name: disk0
+```
+
+A `VirtualMachine` named 'example-decentralized-lm-vm' is created, utilizing a `DataVolume` named 'example-decentralized-lm-vm-dv'. This `DataVolume` can contain any kind of disk image, for example fedora. Due to the `runStrategy` being set to `Always`, the virtual machine will always start once the YAML is applied to the cluster.
+
+Within the target cluster, all necessary resources must be available to deploy the target virtual machine. To hold the contents of the source disk, a blank `DataVolume` is created. While not mandatory, it is advisable to retain the original name.
+
+Target DataVolume:
+
+```yaml
+apiVersion: cdi.kubevirt.io/v1beta1
+kind: DataVolume
+metadata:
+  annotations:
+	cdi.kubevirt.io/storage.usePopulator: "true"
+  name: example-decentralized-lm-vm-dv
+spec:
+  source:
+	blank: {}
+  storage:
+    storageClassName: <other storage class>
+    resources:
+      requests:
+        storage: 10Gi
+```
+
+Target Virtual Machine:
+
+```yaml
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  annotations:
+	kubevirt.io/restore-run-strategy: Always
+  name: example-decentralized-lm-vm
+spec:
+  runStrategy: WaitAsReceiver
+  template:
+	metadata:
+  	  creationTimestamp: null
+	spec:
+  	  architecture: amd64
+  	  domain:
+    	    devices:
+    	    autoattachGraphicsDevice: false
+    	  disks:
+    	  - disk:
+             bus: virtio
+      	    name: disk0
+    	  interfaces:
+    	  - masquerade: {}
+      	    name: default
+    	  machine:
+      	    type: q35
+    	  resources:
+      	    requests:
+             memory: 128Mi
+	networks:
+	- name: default
+  	  pod: {}
+	volumes:
+  	  - dataVolume:
+      	    name: example-decentralized-lm-vm-dv
+    	  name: disk0
+```
+
+The source and target virtual machine definitions differ in two key aspects:
+
+* The target's `runStrategy` is set to `WaitAsReceiver`.
+* The target virtual machine includes an annotation specifying the `runStrategy` after migration, which should be the same as the source, in this case it is `Always`.
+
+Any additional resources beyond the virtual machine disk, such as `Secret`s or `Configmap`s, must also be created on the target cluster. Upon creation of the `VirtualMachine` resource, KubeVirt will generate a corresponding receiving `VirtualMachineInstace`, which can then be referenced in the target migration resource.
```

Comment:
maybe this is a good chance to highlight that an orchestration layer outside kubevirt to ensure these resources exist in the source cluster, and that is a future goal (is it?..)

Is there any ongoing work about this orchestration layer ? The network team has some requirements for it :)